3.9-minimal/Dockerfile.rhel8

```diff
@@ -41,33 +47,38 @@ LABEL summary="$SUMMARY" \
       com.redhat.component="python-39-container" \
       name="ubi8/python-39" \
       version="1" \
-      usage="s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=3.9/test/setup-test-app/ ubi8/python-39 python-sample-app" \
+      usage="s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=3.9/test/setup-test-app/ ubi8/python-39-minimal python-sample-app" \
       com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI" \
-      io.buildpacks.stack.id="com.redhat.stacks.ubi8-python-39" \
+      io.buildpacks.stack.id="com.redhat.stacks.ubi8-python-39-minimal" \
       maintainer="SoftwareCollections.org <sclorg@redhat.com>"
 
-RUN INSTALL_PKGS="python39 python39-devel python39-setuptools python39-pip nss_wrapper \
-        httpd httpd-devel mod_ssl mod_auth_gssapi mod_ldap \
-        mod_session atlas-devel gcc-gfortran libffi-devel libtool-ltdl enchant" && \
-    yum -y module enable python39:3.9 httpd:2.4 && \
-    yum -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
+# Very minimal set of packages
+# Python is obvious in the Python container :)
+# glibc-langpack-en is needed to set locale to en_US and disable warning about it
+# findutils - find command is needed for fix-permissions script
+# nss_wrapper - used in generate_container_user script
+RUN INSTALL_PKGS="python39 glibc-langpack-en findutils nss_wrapper" && \
+    microdnf -y --setopt=tsflags=nodocs --setopt=install_weak_deps=0 install $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
-    # Remove redhat-logos-httpd (httpd dependency) to keep image size smaller.
-    rpm -e --nodeps redhat-logos-httpd && \
-    yum -y clean all --enablerepo='*'
+    microdnf -y clean all --enablerepo='*'
 
 # Copy the S2I scripts from the specific language image to $STI_SCRIPTS_PATH.
-COPY 3.9/s2i/bin/ $STI_SCRIPTS_PATH
+COPY 3.9-minimal/s2i/bin/ $STI_SCRIPTS_PATH
 
 # Copy extra files to the image.
-COPY 3.9/root/ /
+COPY 3.9-minimal/root/ /
 
 # Python 3 only
 # Yes, the directory below is already copied by the previous command.
 # The problem here is that the wheels directory is copied as a symlink.
 # Only if you specify symlink directly as a source, COPY copies all the
 # files from the symlink destination.
 COPY 3.9/root/opt/wheels /opt/wheels
+
+# Also creates the directory - it has to be done here so the latter
+# fix-permissions fixes this directory as well.
+WORKDIR ${HOME}
```

Comment:
Because it not just set the directory as your workdir but _also_ creates it if it does not exists yet. For me, it was new and I have to move the line before fix-permissions so I've also documented the reason. Should I somehow rephrase it?

Reply:
Oooh, now I understand. I thought the "also" referred to some previous commands, or something like that.

How about "This command sets (and also creates if necessary) the home directory", or something like that?

Reply:
Fixed, thank you.